pipelines/upgrade/07-server_to_final.yaml

```diff
@@ -20,3 +20,11 @@
       scenario_version: "{{ forklift_upgrade_version_final }}"
     - foreman_server_repositories
     - foreman_installer
+  post_tasks:
+    - name: Run remove Pulp 2
+      command: foreman-maintain content remove-pulp2
```

Comment:
i think this will stop and ask if you want to proceed, in retrospect we should have added a --force option

Maybe you force some input into it?

Reply:
Ahh yea, I see that here -- https://github.com/theforeman/foreman_maintain/pull/442/files#diff-69a7be004ceb14fd5a9c0c117906b77e5dc900604a03df595ae737d2484b9bfbR42

I will test a way around this to run it hands off.

Reply:
Should it be `shell: yes | foreman-maintain content remove-pulp2` while we file an issue for a `--yes` or `--force`? (I feel like `--force` may be too strong for just a "do not ask for confirmation").

Reply:
Yeah, I too hate it when `--force` means `--assume-yes` (or similar).

Reply:
yes, i'd prefer --assume-yes, more meant a general option